packages/datadog_flutter_plugin/lib/src/web_helpers.dart

```diff
@@ -52,3 +53,30 @@ dynamic valueToJs(Object? value, String parameterName) {
   throw ArgumentError(
       'Could not convert ${value.runtimeType} to javascript.', parameterName);
 }
+
+String? convertWebStackTrace(StackTrace? stackTrace) {
+  if (stackTrace == null) return null;
+
+  final dartLineRegex =
+      RegExp(r'(?<file>.+) (?<location>\d+:\d+)\s*(?<function>.+)');
```

Comment:
should we extract it a static field or simply class property? it is same expression everytime, so maybe we can create the object once and compile the expession, no need to create another one with each method invocation.

Reply:
Yes, good call. I originally tried to make it `const` but apparently RegExp doesn't have a const constructor.